### PR TITLE
PMM-4177 Add PostgreSQL metrics.

### PR DESCRIPTION
**Pull request by BupycHuk**

### CODE REVIEW

File: api/qanpb/collector.proto [hunk 1]
```diff
@@ -295,6 +295,34 @@ message MetricsBucket {
...
+  // Total number of local blocks written by the statement.
+  float m_local_blks_written = 207;
+  // Total number of temp blocks read by the statement.
+  float m_temp_blks_read = 208;
+  // Total number of temp blocks written by the statement.
+  float m_temp_blks_written = 209;
+  // Total time the statement spent reading blocks, in milliseconds (if track_io_timing is enabled, otherwise zero).
+  float m_blk_read_time = 210;
+  // Total time the statement spent writing blocks, in milliseconds (if track_io_timing is enabled, otherwise zero).
+  float m_blk_write_time = 211;
```
Comment (AlekSi) [inline api/qanpb/collector.proto]: Let's start using this branch for now, and discuss in Minsk:
1. if we want to use embedded messages there;
2. if we want to split pmm-agent<->pmm-managed and pmm-managed<->pmm-agent protocols.

Comment (BupycHuk) [reply api/qanpb/collector.proto]: ok

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-389/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-389/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-389/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-389/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-389/

**Pull request merged**